### ISSUE TITLE
Create a MtG card component

### ISSUE THREAD

**Issue by davidyell** [davidyell/Learning-React-MTG-Frontend#7]
Need to create an actual card component which takes some card data and tries to accurately represent an actual magic card.

Example payload for a card

```json
                "card": {
                    "id": 23,
                    "border_color": "black",
                    "color_identity": "W",
                    "color_indicator": null,
                    "colors": "W",
                    "converted_mana_cost": 2,
                    "flavor_name": null,
                    "flavor_text": "\"I thought of fate as an iron lattice, intricate but rigidly unchangeable. That was until some force bent fate's bars to spare my life.\"\n—Ilromov, traveling storyteller",
                    "keywords": null,
                    "layout": "normal",
                    "mana_cost": "{1}{W}",
                    "mana_value": 2,
                    "multiverseId": "132105",
                    "name": "Ghost Warden",
                    "power": "1",
                    "rarity": "common",
                    "set_code": "10E",
                    "subtypes": "Spirit",
                    "supertypes": null,
                    "text": "{T}: Target creature gets +1/+1 until end of turn.",
                    "toughness": "1",
                    "type": "Creature — Spirit",
                    "types": "Creature",
                    "uuid": "d68306e2-9877-5987-84b3-12b8234c8eec"
                },
                "meta": {
                    "quantity": 9,
                    "is_sideboard": false
                }
```

You might consider trying out Storybook to build this? https://storybook.js.org/

**Comment by davidyell** [comment]
Resolves #1

**Comment by davidyell** [comment]
Resolved in https://github.com/davidyell/Learning-React-MTG-Frontend/pull/10
